tedbot/tedbot.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: tedbot-pv
+  labels:
+    identifier: tedbot
+spec:
+  capacity:
+    storage: 1Gi
+  volumeMode: Filesystem
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  mountOptions:
+    - hard
+    - nfsvers=3
+  nfs:
+    path: /kubernetes-test/tedbot
+    server: 192.168.1.42
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: tedbot-pv-claim
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
+  selector:
+    matchLabels:
+      identifier: tedbot
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: tedbot
+spec:
+  schedule: "0 12 * * *"
```

Comment:
i use `0 12 * * 1-5` to get weekdays only, but it's your call at the end of the day

Reply:
👌 I needed this.